.vscode/tasks.json

```diff
@@ -16,6 +16,132 @@
         "isDefault": true
       }
     },
+    {
+      "label": "Build neovim extension only",
+      "dependsOn": [
+        "neovim start",
+        "ESBuild-neovim-registry",
+        "ESBuild-cursorless-neovim"
+      ],
+      "group": "build"
+    },
+    {
+      "label": "Build neovim extension tests only",
```

Comment:
not sure "only" means anything here?

Reply:
the same as what is used for building the extension only:
```json
    {
      "label": "Build extension only",
      "dependsOn": ["Populate dist", "ESBuild"],
      "group": "build"
    },
```

But I'm happy to remove the "only" in both cases above if you prefer.